### ISSUE TITLE
Update BluedogDB's Metadata

### ISSUE THREAD

**Issue by SofieBrink** [KSP-CKAN/NetKAN#10247]
One of BDB's part modules has moved house to KSPCommunityPartModules.
This means that for the next public release `v1.14.1` BDB will have to depend on KSPCommunityPartModules `v0.2.0` or above. 
It is unclear when exactly this release is to be expected.

![image](https://github.com/user-attachments/assets/8b627ce5-872d-48ea-8326-6def34b7a361)
![image](https://github.com/user-attachments/assets/9025f4f6-7f82-45f3-8e3c-995220209a39)

Also see the pull request on BDB's end [here](https://github.com/CobaltWolf/Bluedog-Design-Bureau/pull/1570)

**Comment by HebaruSan** [comment]
Thanks for letting us know. I've subscribed to release notifications for the BDB repo and will merge #10248 upon receipt of same.

**Comment by SofieBrink** [comment]
Fantastic, thanks.

One question though, is specifying a min_version for the KSPCommunityPartModules dep not desirable for some reason?
I feel like with the current setup, if a user has `v0.1.1` installed and updates BDB without updating KSPCPM CKAN will be satisfied even though it won't work as intended with that version of the mod.

**Comment by HebaruSan** [comment]
Good point, will add that.

**Comment by HebaruSan** [comment]
Hey @SofieBrink, is this going to happen soon?

**Comment by HebaruSan** [comment]
OK, please let us know if you make progress with them, and we can re-open this stuff then. Thanks!

**Comment by zorg2044** [comment]
Oh if this is waiting for our next release we don't really have a specific timeline yet. Its probably not within the next few days though. Perhaps a month or so while we're working on some  stuff. Will ping you when we're ready.

**Comment by HebaruSan** [comment]
Yeah, if we merge this before then, then the _current_ release that still has the module in it would receive these changes, which would probably mess everything up if I'm not mistaken?

**Comment by SofieBrink** [comment]
It shouldn’t really matter, the new module has a different name. It’d just be a useless dependency. But that decision isn’t up to me
